lib/aullar/view.moon

```diff
@@ -793,8 +795,11 @@ View = {
     pos = @position_from_coordinates(event.x, event.y)
     if pos
       @cursor\move_to :pos, extend: true
-    elseif event.y < 0
-      @cursor\up extend: true
+    elseif event.y < 10
```

Comment:
I saw the discussion of this on the issue, and yes, having a hard coded `10` here isn't ideal. The explanation for this is the `margin`, so the original code should have been `event.y <= @margin` I believe.